onnxscript/values.py

```diff
@@ -213,6 +213,8 @@ def adapt(input: ExtendedModeValue) -> EagerModeValue:
             return input
         elif isinstance(input, (bool, int, float)):
             return tensor.Tensor(np.array(input))
+        elif isinstance(input, str):
```

Comment:
This happens because approximate was passed in positionally. We should create a wrangler instead until #305 is fixed

Reply:
Is there any "str" type signature that I can reference?

Reply:
strs can be attributes last time I checked

Reply:
Now that I think about it again, does having an if condition on a str make it an input? So this change is desired. @gramalingam